src/components/Layouts/ReportLayout.js

```diff
@@ -75,14 +77,20 @@ class ReportLayout extends Component {
             if (maxHeight < item.element.clientHeight) {
               maxHeight = item.element.clientHeight;
             }
-            item.element.style.top = `${accumulatedHeight}px`;
+            if (isAutoHeightLayout) {
```

Comment:
Default is false now right? This will change existing behavior  when not passing auto height true.

Reply:
yeah default is false, while investigation reports pass true.

Reply:
I'm talking about other users who use this.. the behavior will be changed for them.